packages/components/src/color-picker/styles.ts

```diff
@@ -112,7 +112,7 @@ export const ColorfulWrapper = styled.div`
 `;
 
 export const DetailsControlButton = styled( Button )`
-	&&&& {
+	&&&&& {
```

Comment:
Why was this change necessary? Was this broken also before moving the component to the `src` folder?